app/konnect/configure/servicehub/manage-services.md

```diff
@@ -1,152 +1,218 @@
     redirects to the Service's overview page.
 
 ### Update a Service
 
-1. From the left navigation menu, click **Services**.
-
-2. Select a Service from the list.
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
 
-3. Edit the Service name and description directly on this page: click on either
+1. Edit the Service name and description directly on this page: click on either
 element to reveal a text box, enter the new text, then click outside of the text
 box to save.
 
-### Delete a Service
+### Share a Service
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
+
+1. Click **Share Service**.
+
+1. Select a user or team to share the Service with.
+
+1. Select a role to grant to the user or team.
 
-1. From the left navigation menu, click **Services**.
+    You can share any Service that you have access to, with
+    the same role or lesser than what you have yourself. An organization admin
+    can share any Service with any role. For more information, see [
+    Manage Teams, Roles, and Users](/konnect/org-management/teams-roles-users/#entity-and-role-sharing).
+
+1. Click **Share Service** to save.
+
+### Delete a Service
 
-2. Select a Service from the list.
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
 
-3. In the top right of the overview page, click the **Actions** menu and select
+1. In the top right of the overview page, click the **Actions** menu and select
 **Delete Service**.
 
-4. In the dialog that appears, confirm that you want to delete this service.
+1. In the dialog that appears, confirm that you want to delete this service.
 
-## Service Versions and Implementations
+## Service versions
 
-### Create a New Service Version
+Every Konnect Service version is associated with one runtime group.
+Any configurations for the Service version, such as implementations, plugins,
+and routes, will also be associated with the same runtime group.
 
-1. From the left navigation menu, click **Services**.
+If a Service has multiple Service versions, each Service version can be
+associated with a different runtime group, or with the same runtime group.
+Through its versions, a Service can made be available in multiple environments,
+simply by assigning Service versions to a different runtime groups.
 
-2. Select a Service from the dropdown menu.
+A common use case for this is environment specialization. For example, if you
+have three runtime groups for `development`, `staging`, and `production`, you
+can manage which environment the Service is available in by assigning that group
+at creation time. So, you might create a v0 in `development`, then a v0.1 in
```

Comment:
Changing the version between RG's doesn't make sense to me. I'd view it more like promotion:

> You might have v1 running in `production`, and be actively working on v2 in `development`. Once it's ready to test, you'd create v2 in `staging` before finally creating v2 in `production` alongside `v1`